### ISSUE TITLE
Misleading Filter Examples

### ISSUE THREAD

**Issue by jameseccles** [MicrosoftDocs/OfficeDocs-o365seccomp#909]
Multiple examples of filters in this article include a mailbox and site query in the same filter. Note it is only possible to have either site OR mailbox queries in a single filter.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a6f37089-3459-9e8e-f336-cb3bd236601f
* Version Independent ID: 3e595e12-d834-fb8d-a7c0-efb2931cd82e
* Content: [Set up compliance boundaries for eDiscovery investigations in Office 365](https://docs.microsoft.com/en-us/office365/securitycompliance/set-up-compliance-boundaries)
* Content Source: [SecurityCompliance/set-up-compliance-boundaries.md](https://github.com/microsoftdocs/officedocs-o365seccomp/blob/public/SecurityCompliance/set-up-compliance-boundaries.md)
* Service: **o365-seccomp**
* GitHub Login: @markjjo
* Microsoft Alias: **markjjo**

**Comment by scanum** [comment]
@jameseccles  Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

@officedocsbot assign @scanum

**Comment by markjjo** [comment]
@jameseccles As per offline discuss with engineering, we determined that the filters list syntax in the example filters is correct. Per your suggestion, I added this note in this section of the topic to highlight/explain the syntax: https://docs.microsoft.com/en-us/office365/securitycompliance/set-up-compliance-boundaries#step-4-create-a-search-permissions-filter-to-enforce-the-compliance-boundary

"The syntax for the Filters parameter includes a filters list. A filters list is a filter that includes a mailbox filter and a site filter separated by a comma. In the previous example, notice that a comma separates Mailbox_ComplianceAttribute and Site_ComplianceAttribute: -Filters "Mailbox_<ComplianceAttribute> -eq '<AttributeVale> '", "Site_ComplianceAttribute -eq '<AttributeValue>' -or Site_Path -like '<SharePointURL>*'". When this filter is processed during the running of a content search, two search permissions filters are created from the filters list: one mailbox filter and one site filter. An alternative to using a filters list would be to create two separate search permissions filters for each agency: one search permissions filter for the mailbox attribute and one filter for the site attributes. In either case, the results will be the same. Using a filters list or creating separate search permissions filters is a matter of preference.
